test/system/post_index_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
```

Comment:
- [ ] **[OPTIONAL]** I think this folder should be in the `spec/` folder where you have other spec files, instead of `test/`